client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -51,6 +51,31 @@ export default React.createClass( {
 		return this.props.domains || [ this.props.domain ]
 	},
 
+	wrongNSMappedDomains() {
+		debug( 'Rendering wrongNSMappedDomains' );
+		const wrongMappedDomains = this.getDomains().filter( domain =>
+			domain.type === domainTypes.MAPPED && ! domain.hasZone );
+
+		let learnMoreLink = ( <a
```

Comment:
It looks like we don't need this `learnMoreLink` variable. It's only used once below.